proto/p4/p4runtime.proto

```diff
@@ -136,6 +136,14 @@ message TableEntry {
   // this is stored, as long as it is returned with the rest of the entry in
   // a Read RPC.
   uint64 controller_metadata = 5;
+  // meter_config and counter_data are convenience fields that enable the
+  // controller to configure the direct resources associated with the table at
+  // the same time as a match-action entry is inserted or modified. When
```

Comment:
Sounds good to me. So we have:
1. the default configuration for a meter is both rates and both burst sizes set to -1, which implies all packets are tagged GREEN.
2. when an entry is added to a table with a direct meter array and no configuration is provide, the default configuration will be used to program the hardware and it will be the configuration returned when doing a Read operation.
3. similarly, for an indirect meter array, all meters in the array will be initialized to the default configuration.
4. Any other meter configuration where one of the rates or one of the burst sizes is a negative number is illegal and will return an error.
5. Any meter configuration where the peak information rate is not greater than or equal to the committed information rate is an error.